mlrun/datastore/v3io.py

```diff
@@ -94,13 +93,13 @@ def get_storage_options(self):
         )
         return self._sanitize_storage_options(res)
 
-    def _upload(self, key: str, src_path: str, max_chunk_size: int = ONE_GB):
+    def _upload(self, key: str, src_path: str, max_chunk_size: int = MAX_CHUNK_SIZE):
         """helper function for upload method, allows for controlling max_chunk_size in testing"""
         container, path = split_path(self._join(key))
         file_size = os.path.getsize(src_path)  # in bytes
         if file_size <= ONE_MB:
             with open(src_path, "rb") as source_file:
-                data = source_file.read()
+                data = memoryview(source_file.read())
```

Comment:
I don't see a reason to keep the 'if file_size <= ONE_MB' case. The second case does exactly the same now